CHANGELOG.md

```diff
@@ -17,6 +17,8 @@
 - PR #5364 Validate array interface during buffer construction
 - PR #5418 Add support for `DataFrame.info`
 - PR #5425 Add Python `Groupby.rolling()`
+- PR #5359 Add duration types 
```

Comment:
duplicate. merging issue?

Reply:
Good catch. Not sure what's happening there.